### PR TITLE
[FR] Update `Internet Relay Chat`

### PR DESCRIPTION
**Pull request by Krystool**
Small corrections.

### CODE REVIEW

File: wiki/Internet_Relay_Chat/fr.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Internet Relay Chat
 
-*Pour plus d'informations, voir : [Internet Relay Chat (Wikipédia)](https://fr.wikipedia.org/wiki/Internet_Relay_Chat)*
+*Pour plus d'informations, lire : [Internet Relay Chat (Wikipédia)](https://fr.wikipedia.org/wiki/Internet_Relay_Chat)*
```
Comment (DoryanR) [inline wiki/Internet_Relay_Chat/fr.md:3]:
```suggestion
*Pour plus d'informations : [Internet Relay Chat (Wikipédia)](https://fr.wikipedia.org/wiki/Internet_Relay_Chat)*
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
The rest is good otherwise

**Comment by peppy** [comment]
Please update your pull request title to imply what file/section you're changing.

**Comment by Krystool** [comment]
> Please update your pull request title to imply what file/section you're changing.

Solved @peppy

**Review by DoryanR — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**